opencti-platform/opencti-graphql/src/modules/entitySetting/entitySetting-types.ts

```diff
@@ -34,6 +34,7 @@ export interface BasicStoreEntityEntitySetting extends BasicStoreEntity {
   platform_hidden_type: boolean;
   enforce_reference: boolean;
   attributes_configuration?: string;
+  workflow_configuration: boolean;
```

Comment:
I don't really understand how workflow_configuration is set. It is not saved in the database, how do we use it exactly ?

Reply:
I believe it is saved in the database as we initialize the entity settings on the startup of the platform (initialization.js:platformInit.js -> **initCreateEntitySettings()**)

Reply:
I still can't see it in my database. Do you ? @yassine-ouaamou 

From what I can see, `initCreateEntitySettings` creates entity settings only if they are not already in the database (which won't work for existing platforms, or only if the entity type is new). 
And it creates the entitySetting based on `defaultEntitySetting`, which doesn't have `workflow_configuration`.

Reply:
You are right! It is not on the database.